docs/includes/apiargs-MongoDBCollection-method-watch-option.yaml

```diff
@@ -70,6 +70,8 @@ optional: true
 source:
   file: apiargs-common-option.yaml
   ref: session
+post: |
+  .. versionadded:: 1.3
```

Comment:
Do you need to specify this here even though every option here was added in version 1.3 since Watch is new?

Reply:
Definitely not. This was likely the result of an over aggressive search/replace.